lib/elastic/enterprise-search/client.rb

```diff
@@ -0,0 +1,92 @@
+require 'set'
+require 'elastic/enterprise-search/configuration'
+require 'elastic/enterprise-search/request'
+require 'elastic/enterprise-search/utils'
+
+module Elastic
+  module EnterpriseSearch
+    # API client for the {Elastic Enterprise API}[https://swiftype.com/enterprise-search].
```

Comment:
I would call it "Elastic Enterprise +Search+ API" and gave a link to the elastic website: https://www.elastic.co/products/enterprise-search